.gitignore

```diff
@@ -1,2 +1,7 @@
 /dist/
 /dist-newstyle/
+/.cabal-sandbox/
+TAGS
```

Comment:
Is this not better in your own global `.gitignore`? I don't know what the current "best practice" is... just a thought.